src/App.tsx

```diff
@@ -1,24 +1,82 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Remove this line, as it is not needed anymore: `/* eslint-disable jsx-a11y/control-has-associated-label */`.